docs/user/contributing/style.md

```diff
@@ -0,0 +1,103 @@
+
+Admonitions should draw the reader’s attention to certain information. Keep admonitions to a minimum, and avoid placing multiple admonitions close to one another. If multiple admonitions are necessary, restructure the information by moving the less-important statements into the flow of the main content. See the [Docusaurus documentation](https://docusaurus.io/docs/markdown-features/admonitions) to learn the correct syntax.
+
+| Type           | Description                                                                                                                                                                                                                               |
+| -------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Note           | Additional guidance or advice that improves product configuration, performance, or supportability                                                                                                                                         |
+| Important      | Advisory information essential to the completion of a task or default configuration settings. Users must not disregard this information.                                                                                                  |
+| Warning/Danger | Information about potential system damage, data loss, or a support-related issue if the user disregards this admonition. <br> If available, offer information to avoid the problem in the future or state where to find more information. |
+| Tip            | Recommendations, suggestions, and alternative methods that might not be obvious. Tips are not essential to using the product.                                                                                                             |
+
+## Code blocks
+
+Code blocks must indicate the programming language of the code snippet.
+
+````md
+```js
+console.log("Hello, world!");
+```
+````
+
+### Commands
+
+- Commands as part of procedures must be inside code blocks with `bash` as the language type. This ensures users get a "copy to clipboard" button.
+- When needed, indicate the directory where users need to move before executing the command.
+
+  ````md
+  ```bash
+  cd  ~/path/to/directory
+  sudo somecommand
+  ```
+  ````
+
+### Common Solus terms
+
+- Solus
+  - Prefer _Solus_ rather than _Solus Project_. Don't use _Solus OS_ or _Solus Linux_.
+  - The names of the different editions of Solus are:
+    - Solus Budgie
+    - Solus GNOME
+    - Solus MATE
+    - Solus Plasma
+- Repositories
+  - Avoid using terms such as _repo_, or _monorepo_, use the specific repository name instead.
+  - Example: "The packages repository", instead of "The monorepo".
+- Dev Tracker
+  - Avoid using the term _Dev Tracker_.
```

Comment:
Instead, use the name of the issue tracker.
  - Example: "The packages issue tracker"